tests/simulator_test.h

```diff
@@ -0,0 +1,346 @@
+// Copyright 2019 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SIMULATOR_TEST_H_
+#define SIMULATOR_TEST_H_
+
+#include <complex>
+
+#include "gtest/gtest.h"
+
+#include "../lib/fuser.h"
```

Comment:
Do you need an include for the simulator you want to use here ?

Reply:
No, I don't need an include for the simulator I want to use here. Includes for the simulators I want to use are in simulator_*_test.cc.